### ISSUE TITLE
[Tracking]  v1 Mainnet Release Requirements

### ISSUE THREAD

**Issue by drwasho** [OpenBazaar/OpenBazaar-Server#287]
These are the following issues, group according to categories, that need to be resolved before we can make officially launch v1 on the mainnet.

---

### Priority Ranking
:star: :star: :star: **High priority** 
- _Primary issues to commit and test asap_
- Mainnet release will not occur without these issues resolved

:star: :star: **Medium priority**
- _Secondary issues to fix and test_
- Mainnet release possible, not but advisable without these issues resolved

:star: **Low priority**
_Peripheral issues; save for last_
- Mainnet release not ideal, but not too bad without these issues resolved

---

### Reputation

- [ ] #283 Calculate and cache the aggregate rating data for a user's store
  - :star: :star:
- [x] #271 ~~Enable Buyer ratings for disputes that they win~~
  - :star: :star: :star: 
- [x] #270 ~~Modify rating api to optionally include GUID~~
  - :star: :star: :star:

### Listings

- [x] #280 ~~When a moderator is added to a store, resave all listings with the new moderator~~
  - :star: :star: :star: 
- [x] #181 ~~Save unconfirmed txs to db~~
  - :star: :star: :star: 
- [x] #285 ~~Create a Bitcoin conversation API~~
  - :star: :star: :star:
- [x] #286 ~~Republish listings after keywords expire from DHT~~
  - :star: 

### Dispute Resolution

- [X] #273 ~~Moderation Fee is not saved~~
  - :star: :star: :star: 

### Database

- [ ] #186 Option to backup database
  - :star:

**Comment by drwasho** [comment]
Ok, all the 'can't launch without' issues have been resolved for the server.
